### PR TITLE
Support multiple aws accounts

### PR DESCRIPTION
**Pull request by scytacki**
This is based off of work that @btek1211 did to get the ECS cluster and LARA running in a separate AWS account.

In the cluster configuration:
- the vpc, subnets, and lambda bucket have been parameterized.
- the vpc and subnets are exposed as outputs so apps running in the cluster can use these values in their templates

In the LARA configuration:
- a new concordqa ssl cert was added
- the vpc and subnets are pulled from the cluster outputs above
- the location of templates folder is parameterized so we can have different versions of the nested templates when working on changes like this one
- the HostedZoneId for Route 53 is parameterized so it can work both with concord.org and concordqa.org

In the nested LoadBalancer stack:
- the Vpc, Subnets, and HostedZoneId are parameterized
- the DNS record identifier is more unique now

### CODE REVIEW

**Comment by scytacki** [comment]
@dougmartin now that you got more cloudformation experience. Can you take a look at this one?
I'll try to resolve the conflict this morning.

File: lara-ecs.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ Description: lara ecs stack
 Parameters:
   Environment:
     Type: String
-    Default: staging
-    AllowedValues: [staging, production]
+    Default: qa
+    AllowedValues: [qa, staging, production]
     Description: Enter staging or production. Default is staging.
```
Comment (dougmartin) [inline lara-ecs.yml]: This description is now invalid.

Comment (scytacki) [inline lara-ecs.yml]:
```suggestion
    Description: Enter staging or production. Default is qa.
```

Comment (scytacki) [inline lara-ecs.yml]:
```suggestion
     Description: Enter staging or production. Default is staging.
```

Comment (scytacki) [inline lara-ecs.yml]:
```suggestion
     Description: Enter staging or production. Default is qa.
```

File: ecs-cluster.yml [hunk 2]
```diff
@@ -6,19 +6,20 @@ Parameters:
...
     Description: VPC Id of existing VPC
-    AllowedPattern: "^(?:vpc-[0-9a-f]{8}|)$"
+    # QA VpcId: vpc-c116d1bb
     Default: vpc-9c9714f9
   SubnetIds:
-    Type: CommaDelimitedList
-    Description: Comma separated list of existing VPC Subnet Ids
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: List of existing VPC Subnet Ids
+    # QA subnets: "subnet-03540c8937acb1963,subnet-0dc2017d2cfe64e01"
```
Comment (knowuh) [inline ecs-cluster.yml:22]: The subnets in the comments don't match the default value, and we actually have 16 subnets now for 3 EC instances.  

Screen Shot 2021-01-07 at 9 49 40 AM

**Pull request merged**